config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
+  #get '/users/:id', to: 'users#show'
```

Comment:
any reason to leave this here? In general terms we should avoid commiting code as it can add noise on long-term